zenoh-plugin-remote-api/EXAMPLE_CONFIG.json5

```diff
@@ -1,13 +1,16 @@
 {
-  mode: "router",
+  mode: 'router',
   plugins_loading: {
-     enabled: true,
-     search_dirs: ["./target/debug", "~/.zenoh/lib"],
-   },
+    enabled: true,
+    search_dirs: ['./target/debug', '~/.zenoh/lib']
+  },
   plugins: {
     remote_api: {
-        "websocket_port": "10000",
-    },
-  },
+      'websocket_port': '10000',
```

Comment:
I suggest removing the quotes around `websocket_port` to fit the rest of the file style.